src/hooks/useApproveAndWithdraw.ts

```diff
@@ -33,6 +36,19 @@ export function useApproveAndWithdraw(
   const { gasStandard, gasFast, gasInstant } = useSelector(
     (state: AppState) => state.application,
   )
+
+  const POOL = POOLS_MAP[poolName]
```

Comment:
Can you use basicPoolContext pools here?

Reply:
Good callout!